dll/win32/shell32/shlfileop.cpp

```diff
@@ -1219,6 +1219,34 @@ static void destroy_file_list(FILE_LIST *flList)
     HeapFree(GetProcessHeap(), 0, flList->feFiles);
 }
 
+static CStringW try_find_new_name(LPCWSTR szDestPath)
+{
+    CStringW mask(szDestPath);
+    CStringW ext(PathFindExtension(szDestPath));
+
+    // cut off extension before inserting a "new file" mask
+    if (!ext.IsEmpty())
+    {
+        mask = CStringW(mask.Left(mask.GetLength() - ext.GetLength()));
+    }
+    mask = CStringW(mask + _T(" (%d)") + ext);
+
+    CStringW newName;
+
+    // trying to find new file name
+    for (int i = 1; i < 100; i++)
```

Comment:
Oh, and why 1? Seems odd.

Reply:
No, its fine! Otherwise first try generate smth "<Copied file> (0).ext" and this is really odd :)

Reply:
@sanchaez probably means `i = 2` would be better.

Reply:
I see @Getequ, it's fine then.